ethpm_types/utils.py

```diff
@@ -1,4 +1,40 @@
 from hashlib import md5
+from typing import Type
+
+from eth_utils import to_bytes, to_hex
+
+
+class Bytes32(bytes):
```

Comment:
I think this class should change to a min/max size validation, and be renamed

The default min should be 0, and the default max should be `None`, which represents unlimited size